clusters/my-cluster/preprod/simple-app/gitops-lab-app-kustomization.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: kustomize.toolkit.fluxcd.io/v1
 kind: Kustomization
 metadata:
   name: gitops-lab-app
-  namespace: flux-system
+  namespace: simple-app
```

Comment:
Hardcoding the namespace is a good idea to keep track on which ressources is deployed on but don't forget that will be overided by Kustomize and Flux Kustomization

Reply:
I see, trying to find the solution for it

Reply:
Please review again on the targetNamespace value, thanks.